### PR TITLE
Added missed imports.

### PR DESCRIPTION
**Pull request by JavaSaBr**
Added missed imports in shaders.

### CODE REVIEW

File: jme3-core/src/main/resources/Common/MatDefs/Light/Deferred.frag [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#import "Common/ShaderLib/GLSLCompat.glsllib"
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Light/Deferred.frag]: This one is not used/deprecated, I'd not include it in this batch

File: jme3-core/src/main/resources/Common/MatDefs/Light/Deferred.vert [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#import "Common/ShaderLib/GLSLCompat.glsllib"
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Light/Deferred.vert]: This one is not used/deprecated, I'd not include it in this batch

File: jme3-core/src/main/resources/Common/MatDefs/Light/GBuf.frag [hunk 3]
```diff
@@ -1,3 +1,4 @@
+#import "Common/ShaderLib/GLSLCompat.glsllib"
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Light/GBuf.frag]: This one is not used/deprecated, I'd not include it in this batch

File: jme3-core/src/main/resources/Common/MatDefs/Light/GBuf.vert [hunk 4]
```diff
@@ -1,3 +1,5 @@
+#import "Common/ShaderLib/GLSLCompat.glsllib"
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Light/GBuf.vert]: This one is not used/deprecated, I'd not include it in this batch

File: jme3-core/src/main/resources/Common/MatDefs/Misc/ColoredTextured.frag [hunk 5]
```diff
@@ -1,3 +1,5 @@
+#import "Common/ShaderLib/GLSLCompat.glsllib"
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Misc/ColoredTextured.frag]: This one is not used/deprecated, I'd not include it in this batch

File: jme3-core/src/main/resources/Common/MatDefs/Misc/ColoredTextured.vert [hunk 6]
```diff
@@ -1,3 +1,5 @@
+#import "Common/ShaderLib/GLSLCompat.glsllib"
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Misc/ColoredTextured.vert]: This one is not used/deprecated, I'd not include it in this batch

File: jme3-core/src/main/resources/Common/MatDefs/Shadow/BasicPostShadow.vert [hunk 7]
```diff
@@ -1,4 +1,5 @@
 #import "Common/ShaderLib/GLSLCompat.glsllib"
+
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Shadow/BasicPostShadow.vert]: mhh... this new line serves nothing

File: jme3-core/src/main/resources/Common/MatDefs/Shadow/PreShadow.frag [hunk 8]
```diff
@@ -1,4 +1,5 @@
 #import "Common/ShaderLib/GLSLCompat.glsllib"
+
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Shadow/PreShadow.frag]: Same, useless newline. Especially when it's the only change in the file

File: jme3-core/src/main/resources/Common/MatDefs/Shadow/PreShadow.vert [hunk 9]
```diff
@@ -1,6 +1,7 @@
 #import "Common/ShaderLib/GLSLCompat.glsllib"
 #import "Common/ShaderLib/Instancing.glsllib"
 #import "Common/ShaderLib/Skinning.glsllib"
+
```
Comment (Nehon) [inline jme3-core/src/main/resources/Common/MatDefs/Shadow/PreShadow.vert]: Same, useless newline. Especially when it's the only change in the file

**Comment by JavaSaBr** [comment]
@Nehon I have updated this :)

**Comment by JavaSaBr** [comment]
@Nehon hm, now I have this problem :)
https://gist.github.com/anonymous/6d5896042c271ff82aae94f7c9751981

**Comment by JavaSaBr** [comment]
@Nehon I have updated this PR, now it works fine on macOS and linux :)

**Pull request merged**